src/node_wasm_web_api.cc

```diff
@@ -0,0 +1,161 @@
+#include "node_wasm_web_api.h"
+
+#include "env-inl.h"
+#include "node_errors.h"
+
+namespace node {
+namespace wasm_web_api {
+
+v8::Local<v8::Function> WasmStreamingObject::Initialize(Environment* env) {
```

Comment:
I think we should add using declarations for these at the top of the file and use those directly.

Reply:
There are very few occurrences of V8 APIs here so I don't think the `v8` namespace makes this less readable. Conversely, I think keeping the `v8` namespace makes the code more approachable, especially in the presence of vague class names such as `Value`. I know that it's inconsistent with most of the existing code base and if you (or anyone else) feels that it should be changed, I'll do so, but I personally don't see it as an improvement right now.

Reply:
> There are very few occurrences of V8 APIs here

There are 54 occurrences in this file (excluding 3 out of the total 57 because those are inside comments) which has 173 lines to consider (excluding the top 8 lines because those are reserved for includes and opening namespaces and the bottom 7 lines because those are for closing namespaces and module registration). So that's like ~30% (54/173) of the number of lines, i.e., we have a `v8::` every 2 lines we skip. Is that still very few? IMO the cutoff is at 10%.

> Conversely, I think keeping the v8 namespace makes the code more approachable, especially in the presence of vague class names such as Value.

Tbh, I quite like including the namespaces (electron does it) because I find it more readable since it tells you where these symbols are coming from but I don't feel strongly about doing that in the context of Node.js because most of the code already specifies using declarations and we haven't faced problems because we don't have to deal with many C++ namespaces coming from our deps, which we do have in electron.

So yes, here my preference would be consistency.

Reply:
> I find it more readable since it tells you where these symbols are coming from

That's exactly my point. Especially with vague names such as `Value`.